### ISSUE TITLE
panic: runtime error with DRONE_STASH_CONSUMER_RSA_STRING

### ISSUE THREAD

**Issue by delfer** [drone/drone#2105]
**Drone 7.3** from **docker** hub
Configured to use Bitbucker server (stash). Privarte key in **DRONE_STASH_CONSUMER_RSA_STRING**

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x958051]

goroutine 1 [running]:
github.com/drone/drone/remote/bitbucketserver.New(0xc420014040, 0x13, 0xc42000c0b9, 0x6, 0xc4200140a9, 0x9, 0xc420014079, 0x14, 0x0, 0x0, ...)
        /go/src/github.com/drone/drone/remote/bitbucketserver/bitbucketserver.go:83 +0x151
github.com/drone/drone/router/middleware.setupStash(0xc4203a6f00, 0xdbcd4a, 0x5, 0x1, 0x0)
        /go/src/github.com/drone/drone/router/middleware/remote.go:90 +0x306
github.com/drone/drone/router/middleware.setupRemote(0xc4203a6f00, 0x20, 0xcfa800, 0x9e2701, 0xc420403d80)
        /go/src/github.com/drone/drone/router/middleware/remote.go:40 +0x22f
github.com/drone/drone/router/middleware.Remote(0xc4203a6f00, 0x13cb580)
        /go/src/github.com/drone/drone/router/middleware/remote.go:21 +0x2f
github.com/drone/drone/extras/cmd/drone/server.server(0xc4203a6f00, 0x0, 0xc4203a6f00)
        /go/src/github.com/drone/drone/extras/cmd/drone/server/server.go:408 +0x161
github.com/drone/drone/vendor/github.com/urfave/cli.HandleAction(0xc9e220, 0xdf5720, 0xc4203a6f00, 0xc4203aa400, 0x0)
        /go/src/github.com/drone/drone/vendor/github.com/urfave/cli/app.go:485 +0xd4
github.com/drone/drone/vendor/github.com/urfave/cli.Command.Run(0xdbdf88, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xdd69c4, 0x1e, 0x0, ...)
        /go/src/github.com/drone/drone/vendor/github.com/urfave/cli/command.go:207 +0xb72
github.com/drone/drone/vendor/github.com/urfave/cli.(*App).Run(0xc4203c6680, 0xc42008a060, 0x2, 0x2, 0x0, 0x0)
        /go/src/github.com/drone/drone/vendor/github.com/urfave/cli/app.go:250 +0x7d0
main.main()
        /go/src/github.com/drone/drone/drone/main_extras.go:43 +0x386
```

**Comment by bunix42** [comment]
I can confirm this error. From my naive point of view the DRONE_STASH_CONSUMER_RSA_STRING is incorrectly handled and causes this nil pointer. 

Using a key file provided by DRONE_STASH_CONSUMER_RSA does not cause any problems as far as I can see.

**Comment by bradrydzewski** [comment]
resolved in 1.0.0-rc.1
